tsconfig.build.json

```diff
@@ -6,6 +6,5 @@
     "outDir": "./dist",
     "rootDir": "./src"
   },
-  "include": ["./src/**/*.ts"],
```

Comment:
Hmm. What is the impact of this being removed? With the `rootDir` set to `./src` this should hopefully prevent anything outside of `./src` from being included, but.... then where would the types from `./types` appear in the final bundle?

Reply:
After thinking about this more, I am fairly certain that relying on ambient declarations in a library is a Bad Idea and will cause all kinds of problems. If you run `yarn build` on #733, for instance, and open up `dist/util.d.ts`, you will see this line:

``` typescript
import { EthQuerySendAsyncFunction } from 'eth-query';
```

Well, as you probably already know, this isn't going to work, because the EthQuerySendAsyncFunction type doesn't exist in `eth-query`. That's something we add, so any attempt to reuse this declaration file in another project will result in errors from TypeScript.

I did some more research and couldn't think of a way to include ambient declaration files (such as the ones we have) in the final build. Maybe it's possible, but maybe there's a good reason why they're excluded, too. So I think it's a better use of time to add types to these packages directly.

Reply:
I did a comparison between this branch and the commit before the `dependencies.d.ts` file was removed, and it was basically the same (just slight sourcemap differences on one file). Certainly no differences in JS or in types. So I'm satisfied that this works.

It looks like triple-slash directives are automatically added to the output type files whenever they are relied upon for the exported types. I expect that you're right, that consumers would be expected to provide these types. Or they would resolve to `any`. That makes sense because the consumer controllers the exact version of the dep used anyway.